CharacterCreation/01_04_Species_Opaleites.txt

```diff
@@ -2,7 +2,18 @@
 Opaleites
 ==============================
 
-  Opaleites are an organic amphibious species that 
+    Opaleites are an organic amphibious species that evolved on a planet that 
```

Comment:
Should we actually name Trioch here?

Reply:
I think we cover it well below. I don't think it fits well here.